### ISSUE TITLE
Getting the plugin to work with custom paths field

### ISSUE THREAD

**Issue by ifiokjr** [spion/typescript-workspace-plugin#3]
I'm just checking to see if it's currently possible to get this plugin working in parallel with a custom paths configuration in the tsconfig.json.

My tsconfig.json is configured as follows.

![image](https://user-images.githubusercontent.com/1160934/47981361-77ec0080-e0c3-11e8-8d8d-16756aac9d03.png)

The configuration in the root package.json is like this. 

![image](https://user-images.githubusercontent.com/1160934/47981441-c7323100-e0c3-11e8-8f05-81a5cdef6cba.png)

Your plugin does everything it promises and now the editor experience for workspace packages is wonderful. However, all my custom paths are broken in the editor (see below for red markers).

![image](https://user-images.githubusercontent.com/1160934/47981510-137d7100-e0c4-11e8-956d-171750f15c3b.png)

I'm wondering if there's a simple workaround, or is it something worth looking into. 

Great plugin by the way!

**Comment by ifiokjr** [comment]
I've created a PR (#4) which I believe solve the issue. The problem I found was that the paths and baseUrl were being overridden by the workspace paths.

@spion Just wondering if you'd also be open to changes to the structure of the code? The plugin solves an important problem, but I found it pretty difficult to first understand and then change the code.

**Comment by ifiokjr** [comment]
Closing as this is no longer relevant.
